src/main.rs

```diff
@@ -693,12 +693,15 @@ fn main() {
 
```

Comment:
I hate github code review so much: it won't let me put a comment on line 690 (`if project_status.branch == None && project_status.files.is_empty() {`)

I think that condition needs to change to take into account ahead/behind: otherwise if you `git checkout aosp/master~` in a repo, it'll get ignored by status.